httpclient5/src/main/java/org/apache/hc/client5/http/auth/KerberosConfig.java

```diff
@@ -81,6 +85,10 @@ public Option getRequestDelegCreds() {
         return requestDelegCreds;
     }
 
+    public String getServiceName() {
```

Comment:
@careylam Please add `@since 5.1` tag to all new public methods introduced in your change-set.

Reply:
I consider this pointless because there no other service class, but `HTTP`.

Reply:
Agree! Ideally we can just hardcode it in the code like what v4.5 does. This code is written just in case there's a plan to support other type. Please suggest. 

Kerberos/SPNego works with these changes. Otherwise, not

Reply:
Hard code it and notice HTTPCLIENT-1625.